DigitalLearningSolutions.Web/Views/NotificationPreferences/_NotificationPreferenceList.cshtml

```diff
@@ -3,7 +3,7 @@
 
 <link rel="stylesheet" href="@Url.Content("~/css/myAccount/notificationPreferences.css")" asp-append-version="true">
 
-<dl class="nhsuk-grid-column-full nhsuk-summary-list preference-list">
+<div class="nhsuk-grid-column-full nhsuk-summary-list preference-list">

```

Comment:
Re HTML markup for screen readers, should this be key-value (dt - dd) combination? The key would be the name of the notification with the description and the value is its status.. 🤔 

https://service-manual.nhs.uk/design-system/components/summary-list

Reply:
Doing this does remove the whole list reading stuff, but it also isn't really an NHS summary list any more. If we want to keep the dl/dt/dd structure, will need to do a whole load of styling changes, simply changing the 2 child divs to a dt/dd pair causes all the styling to break

Reply:
I think we should use the accessible markup where-ever possible, so we should probably give this change a go! But maybe timebox this (keep at 1h max)?